hr_skill/models/hr_skill.py

```diff
@@ -50,3 +56,23 @@ def name_get(self):
                 current = current.parent_id
             res.append((skill.id, ' / '.join(reversed(names))))
         return res
+
+    @api.onchange('free_text')
+    def _onchange_free_text(self):
+        if self.is_free_text_field:
+            hr_skill_obj = self.env['hr.skill']
+            hr_skill_obj.create({
+                'name': self.free_text,
+                'state': 'pending',
+                'parent_id': self._origin.id,
+            })
```

Comment:
Hmmmm i think this is a bad idea.. onchange gets executed very often, like in, any time you change the free_text field. So many skill records will be created, when in fact, you want the "create" action for the new skill to be done only once. So I think it's better to go the way I said on GitLab, remove the onchange function and instead inherit the "write" function of `hr.skill` and check for "free_text" to be in vals. If yes, you will add `state=pending` and call super of the write, thereby creating the record.